core/commands/config.go

```diff
@@ -58,6 +58,14 @@ Set the value of the 'datastore.path' key:
 		args := req.Arguments()
 		key := args[0]
 
+		// This is a temporary fix until we move the private key out of the config file
+		switch key {
+		case "Identity", "Identity.PrivKey":
```

Comment:
@jbenet you cant look it up by the lowercase key, but we can add that to the permissions check.